jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAddConnectionDialog.kt

```diff
@@ -158,8 +159,11 @@ open class ToolkitAddConnectionDialog(
                         \t startUrl=${modal.startUrl}
                     """.trimIndent()
                 }
-
-                loginSso(project, startUrl, region, scopes)
+                if (CODEWHISPERER_SCOPES.all { it in scopes }) {
+                    loginSso(null, startUrl, region, scopes)
+                } else {
+                    loginSso(project, startUrl, region, scopes)
+                }
```

Comment:
login behavior should not change specifically for codewhisperer

Reply:
updated